t/api/08-jobtemplates.t

```diff
@@ -786,6 +788,37 @@ $t->post_ok(
     'posting invalid YAML template results in error'
 );
 
+my $template_yaml = <<'EOM';
```

Comment:
Elsewhere in the file we just have multi-line `'` strings... is there an advantage to using this syntax instead?

Reply:
This is how the hard-core Perl fans do it.

Reply:
Just use them, I was just kidding :-)